autopush/tests/test_integration.py

```diff
@@ -183,6 +188,9 @@ def get_notification(self, timeout=0.2):
         except:
             return None
 
+    def get_notification(self, timeout=0.2):
```

Comment:
Yup, the same way __getattribute__ bypasses it, grab the actual one with:
 ``get = object.__getattribute__(self, "get_notification")```